include/image-commands.mk

```diff
@@ -332,6 +333,18 @@ define Build/sysupgrade-tar
 		$@
 endef
 
+define Build/sysupgrade-dtb-ramdisk-tar
+	$(eval dtb_file=$(word 1,$(1)))
+	$(eval ramdisk_file=$(word 2,$(1)))
+	sh $(TOPDIR)/scripts/sysupgrade-tar.sh \
+		--board $(if $(BOARD_NAME),$(BOARD_NAME),$(DEVICE_NAME)) \
+		--kernel $(call param_get_default,kernel,$(1),$(IMAGE_KERNEL)) \
+		--rootfs $(call param_get_default,rootfs,$(1),$(IMAGE_ROOTFS)) \
+		--dtb $(call param_get_default,dtb,$(1),$(dtb_file)) \
+		--ramdisk $(call param_get_default,ramdisk,$(1),$(ramdisk_file)) \
+		$@
+endef
```

Comment:
Do you plan to use this for other devices? If not, you should keep it in target/linux/mpc85xx/image/Makefile.

Reply:
Dropped!